script/testing/oltpbench/test_oltpbench.py

```diff
@@ -23,13 +23,15 @@ def __init__(self, args):
         self.terminals = int(
             self.args.get("terminals", constants.OLTP_DEFAULT_TERMINALS))
         self.loader_threads = int(
-            self.args.get("loader_threads", constants.OLTP_DEFAULT_LOADER_THREADS))
+            self.args.get("loader_threads",
+                          constants.OLTP_DEFAULT_LOADER_THREADS))
         self.time = int(
             self.args.get("client_time", constants.OLTP_DEFAULT_TIME))
         self.weights = str(self.args.get("weights"))
         self.transaction_isolation = str(
             self.args.get("transaction_isolation",
                           constants.OLTP_DEFAULT_TRANSACTION_ISOLATION))
+        self.query_mode = self.args.get("query_mode")
```

Comment:
Should we have a default value for `query_mode `?

Reply:
dont have to, according to the [doc of jdbc], if we do not specify anything,
- the default value for `preferQueryMode` is `extended`
- the default value for `prepareThreshold` is `5`

Reply:
it could be nice to throw in there as a sort of self-documentation. Or maybe throw that info in a comment somewhere

Reply:
i do have documented it in the README.md for the testing script folder (contents as below)

## QueryMode
For both `junit` and `oltpbench`, we support 2 query modes with the optional argument `--query-mode`
- `simple`
- `extended` (default if not specified)

If you specify the `--query-mode extended`, you then can also indicate the prepare threshold (default is `5`) with the optional argument `--prepare-threshold` with type `int`. Please be reminded that if you choose the query mode as `simple`, the prepare threshold will be ignored.